lib/mixlib/install/dist.rb

```diff
@@ -0,0 +1,29 @@
+module Mixlib
+  class Install
+    class Dist
+      # This class is not fully implemented, depending it is not recommended!
+      # Default project name
+      PROJECT_NAME = "Chef".freeze
+      # Binary repository base endpoint
+      PRODUCT_ENDPOINT = "https://packages.chef.io".freeze
+      # Omnitruck endpoint
+      OMNITRUCK_ENDPOINT = "https://omnitruck.chef.io".freeze
+      # Default product name
+      DEFAULT_PRODUCT = "chef".freeze
+      # Default base product page URL
+      PRODUCT_URL = "https://downloads.chef.io".freeze
+      # Default github org
+      GITHUB_ORG = "chef".freeze
+      # Bug report URL
+      BUG_URL = "https://github.com/chef/omnitruck/issues/new".freeze
+      # Support ticket URL
+      SUPPORT_URL = "https://www.chef.io/support/tickets".freeze
+      # Resources URL
+      RESOURCES_URL = "https://www.chef.io/support".freeze
+      # MacOS directory name
+      MACOS_DIR = "chef_software".freeze
+      # Windows directory name
+      WINDOWS_DIR = "opscode".freeze
```

Comment:
I'd prefer is this was something like WINDOWS_INSTALL_DIR so it was more clear in the templates below